### PR TITLE
Readme - do not merge

### PR DESCRIPTION
**Pull request by selvasingh**
prep

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
+This README is based on the released stable version (1.2.0). If you are looking for other releases, see [More Information](#more-information).
 
 The Azure Management Libraries for Java is a higher-level, object-oriented API for managing Azure resources. If you are looking for Java client libraries for Azure services, please see [Azure libraries for Java](https://docs.microsoft.com/en-us/java/azure/java-sdk-azure-install) - this page will be maintained regularly to ensure that it is up to date.
 
 Note: This page will be maintained regularly to ensure that it is up to date.
 
-## Feature Availability and Road Map as of Version 1.1.0 ##
+## Feature Availability and Road Map as of Version 1.2.0 ##
 
 <table>
```
Comment (daschult) [inline README.md]: @selvasingh Why do we do a HTML table here instead of a markdown table? Just curious.

Comment (selvasingh) [inline README.md]: for a bit more control on formatting, nothing else

**Comment by selvasingh** [comment]
@jianghaolu - please update the Maven dependency version number and snapshot version number for Azure Management Libraries for Java

**Comment by selvasingh** [comment]
@jianghaolu  - changed the release version number to 1.2.1 and snapshot number to 1.2.2-SNAPSHOT. However it is unclear if we should continue to point the snapshots to 1.0.5-SNAPSHOT for client runtime, authentication, etc.

**Comment by daschult** [comment]
I updated the snapshots to 1.1.1-SNAPSHOT for client runtime, authentication, etc. after talking to Anu.

**Pull request merged**